### ISSUE TITLE
Received 2 new card readers but 1st one is not working...

### ISSUE THREAD

**Issue by danielbrunt57** [adonno/tagreader#124]
I received my two tag readers yesterday, thank you.  The first one I was able to connect to its wi-fi URL and configured it to connect to my wi-fi and it showed up in HA but it will not scan any tags.  I moved onto the 2nd one and immediately noticed a difference between the two... the first one has no indicator lights after power up whereas the 2nd one lit up with a red LED.  The 1st one is successfully connected to my wi-fi as I can ping its DHCP address but nothing beyond that.  It does emit a sound shortly after being powered up but no red LED and no scans.  I can turn on the blue LED from HA and have noticed the blue LED is somewhat different than the 2nd reader... the first looks like four dots in a circle but the 2nd reader looks like a single bright blue LED.

**Comment by danielbrunt57** [comment]
![IMG_20210714_233230](https://user-images.githubusercontent.com/49846893/125740430-3d708448-2935-4a46-911e-1c77ef43ba40.jpg)
![IMG_20210714_233347](https://user-images.githubusercontent.com/49846893/125740439-531a227d-4803-4c9e-8ee5-724363cbf19f.jpg)

**Comment by danielbrunt57** [comment]
Fixed!

Opened the case and found a bad solder joint on the ground...
![IMG_20210715_165006](https://user-images.githubusercontent.com/49846893/125873490-41e32d67-9d8d-46f9-9489-a77d8538f31f.jpg)

Also found the reason for the difference in the blue LED's...
![IMG_20210715_165526](https://user-images.githubusercontent.com/49846893/125873466-6918ca37-faeb-4fe9-ace0-6071411294ba.jpg)

**Comment by adonno** [comment]
Hey, @danielbrunt57 I am sorry for the poorly soldered GND I usually test them all and take great care in making sure I don't send defective devices. 

I also seem to have sent you TWO Different versions of the LID which is also not standard practice. 
Is there anything I can do?

**Comment by danielbrunt57** [comment]
You can send me a new lid so they are the same.  I carefully dremelled the white plastic away on the 4 dot version so I see a blue circle from inside it now but it still appears as 4 dots which I can see from the back side if I hold it up to a light.

While I got the first one up and running after re-soldering the GND, the green LED is not working.  Blue is working and if I turn that on and then scan, it just turns off rather than changing to green like my 2nd reader does.  Could that be another bad solder joint?

**Comment by nielsrune** [comment]
Just to add to this, my LED isn't working at all.
After seeing this post, I opened the lid which has the hole (equal to the bottom one in OP's picture)

But the LED won't turn on, even when switching on in HA :-(

**Comment by adonno** [comment]
@danielbrunt57   @nielsrune  would you mind hitting me up with an email please ?  so that I don't lose track?

**Comment by nielsrune** [comment]
> @danielbrunt57 @nielsrune would you mind hitting me up with an email please ? so that I don't lose track?

@adonno I beat you to it :-) See my e-mail of August 8th to adonno@adonno.com, subject "LED seems to be faulty or missing"

**Comment by adonno** [comment]
i'm working on a solution by end of the week

**Comment by 0n3-70uch** [comment]
> Just to add to this, my LED isn't working at all.
> After seeing this post, I opened the lid which has the hole (equal to the bottom one in OP's picture)
> 
> But the LED won't turn on, even when switching on in HA :-(

Hello, I have the same problem. The LED worked fine for just one hour...

**Comment by hermanops** [comment]
Hi, I also have this problem. I bought the Tag Reader Sandwich Version and the LED isn't working anymore

**Comment by Strangehadron** [comment]
I've made a tag reader, same wiring than the schematics and same code and the led wasn't working either. (a W2812B)

I've comment the line : `default_transition_length: 10ms `to desactivate it and now i can turn on/off and change the color of my led via HA, but the led don't flash when scan a tag.

Using the last version of esphome : 2021.8

**Comment by nielsrune** [comment]
> i'm working on a solution by end of the week

@adonno, any update on this one?
As per the picture I sent by e-mail, my LED seems physically missing.

**Comment by adonno** [comment]
small update , i have been waiting on parts and they finally arrived i'll update you this weekend

**Comment by 0n3-70uch** [comment]
> small update , i have been waiting on parts and they finally arrived i'll update you this weekend

Great thank you.

**Comment by adonno** [comment]
@danielbrunt57 i am unable to get back to you via email it seems i'm blocked ?

**Comment by adonno** [comment]
everyone should have gotten their fixed devices?

**Comment by danielbrunt57** [comment]
No sign of mine yet...

**Comment by adonno** [comment]
yeah just noticed it's still in transit.  it was sent on Dec 8. 
i will have to check with postal services there are several packages stuck

**Comment by adonno** [comment]
this time they finally arrived. :-D
